lib/betterdoc/containerservice/railtie.rb

```diff
@@ -56,6 +56,14 @@ class ActionView::Base
           end
         end
 
+        # Make sure that external APIs can be initialized here before application starts
+        ActiveSupport.on_load(:before_initialize) do
+          if ENV['MONITOR_INSTRUMENTAL_API_KEY'].present?
+            require 'instrumental_agent'
+            @instrumental_agent ||= Instrumental::Agent.new(ENV['MONITOR_INSTRUMENTAL_API_KEY'])
```

Comment:
Same comment about variable as in Borg PR https://github.com/betterdoc-org/borg/pull/7095#discussion_r334880725